CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contribution Guidelines
+
+Thank you for contributing to the Education Design System! Feel free to reach out in [#help-edu-design-system](https://chanzuckerbergteam.slack.com/archives/CTFV79JH4) or [office hours](https://www.google.com/url?q=https://docs.google.com/spreadsheets/d/1zZguiMQHQLANjfUF-LjmPkbZ29I7ZXfl8TRDAhqDL0o/edit&sa=D&source=calendar&ust=1617083817378000&usg=AOvVaw2MJp29FMPv2AD1WJFX5Q2x) if you have additional questions.
```

Comment:
Since this is public (although I can't imagine too many people are looking at it), we may want to note that these channels/documents are only available to CZI employees.